guides/common/modules/proc_migrating-project-or-smartproxy-using-backup.adoc

```diff
@@ -17,7 +17,7 @@ endif::[]
 . Perform a full backup of your {ProjectServer} or {SmartProxy}.
 This is the source {EL}{nbsp}8 server that you are migrating.
 For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin#Performing_a_Full_Backup_admin[Performing a full backup of {ProjectServer} or {SmartProxyServer}] in _{AdministeringDocTitle}_.
-. Deploy a system with {EL}{nbsp}9.
+. Deploy a {EL}{nbsp}9 system with the same hostname and configuration as the source server.
```

Comment:
Does this contradict https://github.com/theforeman/foreman-documentation/pull/3356/files?

Also, from a practical standpoint, can you have two hosts with the same hostname?

Also, what does "configuration" mean exactly? Can users follow a specific procedure that we can link to?

Reply:
> Does this contradict https://github.com/theforeman/foreman-documentation/pull/3356/files?
No, satellite-clone adjusts the hostname itself (and then does a restore), while a "plain" restore requires the system to have the same hostname as a prerequisite

> Also, from a practical standpoint, can you have two hosts with the same hostname?
`/etc/hostname` is a local thing, as long the system "thinks" it has that name, it's fine.

> Also, what does "configuration" mean exactly? Can users follow a specific procedure that we can link to?
That is the same wording we use in `guides/common/modules/con_restoring-server-or-smart-proxy-from-a-backup.adoc` and `guides/common/modules/proc_restoring-from-a-full-backup.adoc`, but we do not provide guidance how exactly to get to that state, no